Tests/MoyaTests/MoyaProvider+RxSpec.swift

```diff
@@ -220,17 +220,17 @@ final class MoyaProviderRxSpec: QuickSpec {
                     context("the callback queue is provided with the request") {
                         it("invokes the callback on the request queue") {
                             let requestQueue = DispatchQueue(label: UUID().uuidString)
-                            var callbackQueueLabel: String?
+                            let callbackQueueLabel = Atomic<String?>(wrappedValue: nil)
```

Comment:
did we have a race condition as well here?

Reply:
Yeah, TSAN caught this one too :)

Reply:
Yeah, TSAN caught this one too :)